example/simple_fetch_content/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+cmake_minimum_required(VERSION 3.12.4) #TODO replace with generated version
+project(simple_example_with_fetch_content LANGUAGES CXX C Fortran)
+
+# 1) fetch c_bindings from the repository
```

Comment:
Can you add MakeAvailable here too? As this is an example, the alternative seems to be important here.

Reply:
I don't understand. Do you want a comment stating that MakeAvailable should be used with CMake 3.14?

Reply:
Yes